diff --git a/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-pattern.js b/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-pattern.js
index 9ec8894..107d340 100644
--- a/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-pattern.js
+++ b/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-pattern.js
@@ -77,7 +77,7 @@
 /******/ 	__webpack_require__.o = function(object, property) { return Object.prototype.hasOwnProperty.call(object, property); };
 /******/
 /******/ 	// __webpack_public_path__
-/******/ 	__webpack_require__.p = "/styleguide/";
+/******/ 	__webpack_require__.p = "./styleguide/";
 /******/
 /******/
 /******/ 	// Load entry module and return exports
diff --git a/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-viewer.js b/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-viewer.js
index 59f8648..5a8b375 100644
--- a/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-viewer.js
+++ b/node_modules/@pattern-lab/uikit-workshop/dist/styleguide/js/patternlab-viewer.js
@@ -180,7 +180,7 @@
 /******/ 	__webpack_require__.o = function(object, property) { return Object.prototype.hasOwnProperty.call(object, property); };
 /******/
 /******/ 	// __webpack_public_path__
-/******/ 	__webpack_require__.p = "/styleguide/";
+/******/ 	__webpack_require__.p = "./styleguide/";
 /******/
 /******/ 	// on error function for async loading
 /******/ 	__webpack_require__.oe = function(err) { console.error(err); throw err; };
diff --git a/node_modules/@pattern-lab/uikit-workshop/webpack.config.js b/node_modules/@pattern-lab/uikit-workshop/webpack.config.js
index 5f34923..7ea85e7 100644
--- a/node_modules/@pattern-lab/uikit-workshop/webpack.config.js
+++ b/node_modules/@pattern-lab/uikit-workshop/webpack.config.js
@@ -17,7 +17,7 @@ const defaultConfig = {
   buildDir: './dist',
   prod: false, // or false for local dev
   sourceMaps: true,
-  publicPath: '/styleguide/',
+  publicPath: './styleguide/',
 };
 
 module.exports = async function() {
